gui/app/modes/HighFlowNasalCannulaMode.qml

```diff
@@ -9,4 +12,83 @@ Mode {
         color: "white"
         font.pointSize: 38
     }
+
+    Rectangle {
+
+        width: 216; height: 440
+        color: "#05121C"
+        radius: 8
+        anchors {
+            top: parent.top;
+            right: parent.right; rightMargin: 8
+        }
+
+        GridLayout {
+            anchors {
+                top: parent.top
+                horizontalCenter: parent.horizontalCenter
+            }
+            columns: 2
+            columnSpacing: 0
+            rowSpacing: 0
+
+            ParameterDisplay {
+                parameterName: qsTr("Pip")
+                parameterUnit: qsTr("cmH<sub>2</sub>O")
+                parameterValue: "23"
+            }
+
+            ParameterDisplay {
+                parameterName: qsTr("PEEP")
+                parameterUnit: qsTr("cmH<sub>2</sub>O")
+                parameterValue: "4"
+            }
+
+            ParameterDisplay {
+                parameterName: qsTr("Flow")
+                parameterUnit: qsTr("l/min")
+                parameterValue: "58"
+            }
+
+            ParameterDisplay {
+                parameterName: qsTr("RR")
+                parameterUnit: qsTr("b/min")
+                parameterValue: "15"
+            }
+
+            ParameterDisplay {
+                parameterName: qsTr("VT")
+                parameterUnit: qsTr("ml")
```

Comment:
mL

Reply:
The design reference says "ml", should I ignore it?
See:
https://xd.adobe.com/view/606e962c-020f-4b23-7140-53ea2fe6b21a-165f/screen/7f791a1d-21c0-43ba-8f6a-8ae214efb5be/xx-Modes